ansible/ansible.cfg

```diff
@@ -4,3 +4,4 @@ host_key_checking=False
 record_host_keys=False
 ssh.insert_key=False
 roles_path=roles
+ask_sudo_pass=True
```

Comment:
Nice, I am assuming that this is needed for local and for configuration against existing vms?